example/src/domain/machinery/Form.js

```diff
@@ -102,6 +138,22 @@ function LabelAndError({ name, label, children, state }) {
   )
 }
 
+function FieldsetAndError({ label, children, state }) {
+  const { showError, error, invalid, isVisited, isSubmitted, hasFocus } = state
+  return (
+    <fieldset>
+      <legend>{label}</legend>
+      {children}
+      {(
+        (hasFocus || isVisited || isSubmitted) && !invalid ? '✓' :
+        hasFocus ? '-' :
+        (isVisited || isSubmitted) && invalid && 'x'
+      )}
```

Comment:
```suggestion
      {determineIndicator(state)}
```

En dan los in het bestand:

```js
function determineIndicator(state) {
  const { showError, error, invalid, isVisited, isSubmitted, hasFocus } = state
  return (
    (hasFocus || isVisited || isSubmitted) && !invalid ? '✓' :
    hasFocus ? '-' :
    (isVisited || isSubmitted) && invalid && 'x'
   )
}
```

Dan kun je die op beide plekken gebruiken.